manifests/init.pp

```diff
@@ -90,10 +90,12 @@
       action => 'accept',
     }
 
-    firewall { "100_udp_${udp_port}_for_memcached":
-      dport  => $udp_port,
-      proto  => 'udp',
-      action => 'accept',
+    if $udp_port != 0 {
```

Comment:
I think this should be implemented in the parent commit. I think it's better to squash the 2 commits into one, to have a single commit without test failures.